Src/Couchbase.Linq.UnitTests/BucketQueryExecutorEmulator.cs

```diff
@@ -42,6 +42,10 @@ public BucketQueryExecutorEmulator(N1QLTestBase test)
             _test = test;
         }
 
+        public void ConsistentWith(MutationState state)
+        {
+        }
```

Comment:
@jeffrymorris

Yes, it's just a unit test emulator.

Brant

On Saturday, June 18, 2016, Jeffry Morris <notifications@github.com> wrote:

> In Src/Couchbase.Linq.UnitTests/BucketQueryExecutorEmulator.cs
> <https://github.com/couchbaselabs/Linq2Couchbase/pull/143#discussion_r67608568>
> :
>
> > @@ -42,6 +42,10 @@ public BucketQueryExecutorEmulator(N1QLTestBase test)
> >              _test = test;
> >          }
> >
> > +        public void ConsistentWith(MutationState state)
> > +        {
> > +        }
>
> Is this supposed to lack an implementation?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/couchbaselabs/Linq2Couchbase/pull/143/files/e7a22d5e65642086904653168685990804f1f1f5#r67608568>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGyff_arXbUw7bcc5loBtXxQol7R2amwks5qNI-ogaJpZM4I05Xi>
> .
>


-- 
Thanks,
Brant Burnett
CenterEdge Software
(336) 598-5934
http://centeredgesoftware.com